assets/js/hooks/session.js

```diff
@@ -774,9 +774,7 @@ const Session = {
 
   queueFocusedCellEvaluation() {
     if (this.focusedId && this.isCell(this.focusedId)) {
-      this.dispatchQueueEvaluation(() => {
-        this.pushEvent("queue_cell_evaluation", { cell_id: this.focusedId });
-      });
+      this.queueCellEvaluation(this.focusedId, false);
```

Comment:
This triggers a broadcast that eventually allows the cell to handle the evaluation in [`handleDispatchQueueEvaluation`](https://github.com/livebook-dev/livebook/pull/2077/files#diff-9b1c18ab3f8c5bd9f855c1e369be79ea89c537c83b644e31eeb47bdae1b84e46R380).